project.develop.json

```diff
@@ -6,6 +6,6 @@
   "role": "arn:aws:iam::002723143144:role/dev-tx_lambda_function",
   "environment": {},
   "hooks":{
-    "build": "pip install -r requirements.txt -t ."
+    "build": "pip install -r ../../requirements.txt -t ."
```

Comment:
NOTE: This is ../../ because when this hook is ran for the deployment of a given lambda function, the current directory is functions/<lambda-fucntion>/. We want to use the same requirements.txt file for all lambda functions.